### DISCUSSION TITLE
Why bolt.Bucket.FillPercent is set to 0.9?

### DISCUSSION THREAD

**Discussion by cenkalti** [etcd-io/etcd#15438] [Q&A]
I found that the concept is introduced with https://github.com/etcd-io/etcd/commit/d883d5acd3cafede2fb06aca945e3bb69b78fd47 but don't see a reason why it is set to 0.9 instead of 1.0. Doesn't it make sense to set 1.0 for better page utilization?

I see that boltdb compaction method (which is not used by etcd) sets it to 1.0:
https://github.com/etcd-io/bbolt/blob/da2f2a53f6e2f25b215b79db2cd417488ef8e955/compact.go#L61

**Comment by ahrtr** [comment]
Please read https://github.com/etcd-io/etcd/pull/15470#issuecomment-1467379416
